src/conversions/anyhow.rs

```diff
@@ -0,0 +1,41 @@
+use crate::exceptions::PyRuntimeError;
+use crate::PyErr;
+
+impl From<anyhow::Error> for PyErr {
+    fn from(err: anyhow::Error) -> Self {
+        PyRuntimeError::new_err(format!("{:?}", err))
+    }
+}
+
+#[cfg(test)]
+mod test_anyhow {
+    use crate::proc_macro::pyfunction;
+    use crate::py_run;
+    use crate::wrap_pyfunction;
+    use crate::{Python, ToPyObject};
+
+    #[pyfunction]
+    fn produce_ok_result() -> anyhow::Result<String> {
+        Ok(String::from("OK buddy"))
+    }
+
+    #[pyfunction]
+    fn produce_err_result() -> anyhow::Result<String> {
+        anyhow::bail!("error time")
+    }
```

Comment:
It's actually not ideal to use `#[pyfunction]` within unit tests of pyo3's lib, because it relies on `pyo3` being available as in the global namespace. We happen to have a self-dev-dependency on pyo3 but it causes unexpected breakages - see #1817 for more detail.

The point here is that it's probably better to just test that `let x: PyErr = produce_err_result().unwrap_err().into()` works ok, and could check the string contents of the error are as expected.

(If you really want to check the `#[pyfunction]` case, that's good too, but it'll need to go in a separate file in `tests/`.)